packages/manager/apps/container/src/container/nav-reshuffle/sidebar/utils.ts

```diff
@@ -37,13 +38,16 @@ export function initTree(
     ) {
       return all;
     }
+    if (node?.region && ![].concat(node.region).includes(region)) {
```

Comment:
Since we maintain the `node.region` object, wouldn't it be better to just be a `string[]` instead of `string | string[]`, it would make the condition easier to read and to maintain